On aarch64-linux CI:
```
[  5%] Building ASM object CMakeFiles/dynarec_native.dir/src/dynarec/arm64/arm64_lock.S.o
[  5%] Building C object CMakeFiles/native_pass1.dir/src/dynarec/arm64/dynarec_arm64_emit_tests.c.o
[  5%] Building C object CMakeFiles/interpreter.dir/src/emu/x64run67_32.c.o
/build/source/src/dynarec/arm64/arm64_lock.S: Assembler messages:
/build/source/src/dynarec/arm64/arm64_lock.S:125: Error: selected processor does not support `swpal x1,x0,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:145: Error: selected processor does not support `swpal w1,w0,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:165: Error: selected processor does not support `swpalh w1,w0,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:185: Error: selected processor does not support `swpalb w1,w0,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:208: Error: selected processor does not support `casal x2,x1,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:234: Error: selected processor does not support `casal w2,w1,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:261: Error: selected processor does not support `casal x2,x1,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:292: Error: selected processor does not support `casal w2,w1,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:320: Error: selected processor does not support `casal w2,w1,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:374: Error: selected processor does not support `swpal w1,wzr,[x0]'
/build/source/src/dynarec/arm64/arm64_lock.S:408: Error: selected processor does not support `crc32x w0,w0,x3'
/build/source/src/dynarec/arm64/arm64_lock.S:415: Error: selected processor does not support `crc32b w0,w0,w3'
make[2]: *** [CMakeFiles/dynarec_native.dir/build.make:243: CMakeFiles/dynarec_native.dir/src/dynarec/arm64/arm64_lock.S.o] Error 1
```

Will look further into this on my ARM hardware…

--- a/pkgs/applications/emulators/box64/default.nix
+++ b/pkgs/applications/emulators/box64/default.nix
@@ -21,14 +21,21 @@ assert
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "box64";
-  version = "0.3.4";
+  version = "0.3.6";
 
   src = fetchFromGitHub {
     owner = "ptitSeb";
     repo = "box64";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-CY5Emg5TsMVs++2EukhVzqn9440kF/BO8HZGQgCpGu4=";
+    hash = "sha256-Z8r7aonVj7VSifgLKx/L7VRdGNnQtTvS4mjI+2+uPxY=";
   };
+
+  # Setting cpu doesn't seem to work (or maybe isn't enough / gets overwritten by the wrapper's arch flag?), errors about unsupported instructions for target
+  # (this is for code that gets executed conditionally if the cpu at runtime supports their features, so setting this should be fine)
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace-fail 'ASMFLAGS  -pipe -mcpu=cortex-a76' 'ASMFLAGS  -pipe -march=armv8.2-a+fp16+dotprod'
+  '';
 
   nativeBuildInputs = [
     cmake

